correction

Lissa2121 opened the pull request:
можеш глянути на іконки, бо я випадково трохи зайшла туди, куди не слід та наче все повиправляла, але в мене не проявляються іконки. На основному все добре?

Lissa2121 committed:
correction

Lissa2121 committed:
new version

Lissa2121 committed:
new version, id

Bohdan-Potopliak commented on src/css/benefits.css:
```diff
@@ -1,3 +1,108 @@
+    width: 188px;
+    font-size: 16px;
+    letter-spacing: -0.02em;
+    color: #FAFAFA;
```
тут можеш пробувати через var руалізовувати, тільки глянь які саме є в base.css, можеш додати потрібні для своєї секції які часто зустрічаються

Bohdan-Potopliak reviewed (changes requested):
окремо свгшку не можу добавити тільки все разом, так як ти комітилась, тому поки перероби сттилі під те про що ми говорили сьогодні, коментар залишив

Lissa2121 committed:
benefits

я закоментувала рут стилі, можеш їх видалити на осовній сторінці та глянути чи буде працювати. Також я додала ховер, фокус
